tests/api/test_controller_opdsfeed.py

```diff
@@ -193,7 +193,7 @@ def page(cls, **kwargs):
                 "feed",
                 lane_identifier=lane_id,
                 library_short_name=circulation_fixture.db.default_library().short_name,
-                _facets=load_facets_from_request(),
+                _external=True,
```

Comment:
In these tests we can drop the `_facets` parameter for the same reason. But we have to add the `_external=True` parameter because the the old `url_for` function used to implicitly add it:
https://github.com/ThePalaceProject/circulation/blob/6ba5bf6a9d6bcd7b99dc3ff84c69067a17c52a9a/api/controller.py#L396-L399